### ISSUE TITLE
MakeHost error using WSL.

### ISSUE THREAD

**Issue by LordOfDisdain** [bitcoin/bitcoin#25205]
I have followed the steps to compile bitcoin on Windows and I am receiving an error that is syntax in nature. This happened when I use this command in the terminal: make HOST=x86_64-w64-mingw32 in the depends folder. I will attach an image to clarify this issue.
**Expected behavior**

A complete compilation of Bitcoin. 
**Actual behavior**

**To reproduce**

**System information**

I am using Bitcoin Core 23.0

I am using an ASUS x555y latop and my OS is Windows 11

<!--- This is normally the contents of a `debug.log` or `config.log` file. Raw text or a link to a pastebin type site are preferred. --
![New Bitcoin issue](https://user-images.githubusercontent.com/42873006/170217362-23730376-c97b-494b-87c3-9490886897f2.png)
>

**Comment by MarcoFalke** [comment]
Please provide exact steps to reproduce and the exact error message copy pasted in here, otherwise there is nothing we can do.

**Comment by LordOfDisdain** [comment]
> Please provide exact steps to reproduce and the exact error message copy pasted in here, otherwise there is nothing we can do.
I can do so. Here is what I did. I followed the steps that was inside of this link: https://github.com/bitcoin/bitcoin/blob/master/doc/build-windows.md#building-for-64-bit-windows
Then I changed directory to the depends folder in the Bitcoin core program. Once I use the Make Host command for the Windows enviroment, I received the issue that this photo below should better explain.
![New Bitcoin issue](https://user-images.githubusercontent.com/42873006/170219980-aad16e53-1d49-416b-9323-3e600cabc20a.png)
